tests/bgp/conftest.py

```diff
@@ -220,29 +220,41 @@ def _setup_interfaces_t1(mg_facts, peer_count):
             _subnets = ipaddress.ip_network(u"10.0.0.0/24").subnets(new_prefix=subnet_prefixlen)
             subnets = (_ for _ in _subnets if _ not in used_subnets)
 
+            loopback_ip = None
+            for intf in mg_facts["minigraph_lo_interfaces"]:
+                if netaddr.IPAddress(intf["addr"]).version == 4:
+                    loopback_ip = intf["addr"]
+                    break
+                if not loopback_ip:
+                    pytest.fail("ipv4 lo interface not found")
+
             for intf, subnet in zip(random.sample(ipv4_interfaces + ipv4_lag_interfaces, peer_count), subnets):
                 conn = {}
                 local_addr, neighbor_addr = [_ for _ in subnet][:2]
                 conn["local_intf"] = "%s" % intf
                 conn["local_addr"] = "%s/%s" % (local_addr, subnet_prefixlen)
                 conn["neighbor_addr"] = "%s/%s" % (neighbor_addr, subnet_prefixlen)
+                conn["loopback_ip"] = loopback_ip
                 if intf.startswith("PortChannel"):
                     member_intf = mg_facts["minigraph_portchannels"][intf]["members"][0]
                     conn["neighbor_intf"] = "eth%s" % mg_facts["minigraph_port_indices"][member_intf]
+                    conn["namespace"] = mg_facts["minigraph_portchannels"][intf]["namespace"]
                 else:
                     conn["neighbor_intf"] = "eth%s" % mg_facts["minigraph_port_indices"][intf]
                 connections.append(conn)
 
             for conn in connections:
                 # bind the ip to the interface and notify bgpcfgd
-                duthost.shell("config interface ip add %s %s" % (conn["local_intf"], conn["local_addr"]))
+                namespace = '-n {}'.format(conn["namespace"]) if conn["namespace"] else ''
```

Comment:
if we are planning to use loopback ip for bgpmon why we need to configure local address ip address and also neighbor address on ptf ?  Is there any other use case other then bgpmon